### PR TITLE
Changed md to rst format #7062

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,6 +3,19 @@ Nereid Checkout
 
 .. image:: https://secure.travis-ci.org/openlabs/nereid-checkout.png?branch=develop
   :target: https://travis-ci.org/openlabs/nereid-checkout
+  :alt: Build Status
```
Comment (PritishC) [inline README.rst]: Remove blank lines where not required.

**Comment by PritishC** [comment]
Rebase.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Nereid Checkout
+===============
+
+.. image:: https://secure.travis-ci.org/openlabs/nereid-checkout.png?branch=develop
+  :target: https://travis-ci.org/openlabs/nereid-checkout
+  :alt: Build Status
+.. image:: https://coveralls.io/repos/openlabs/nereid-checkout/badge.png?branch=develop
```
Comment (PritishC) [inline README.rst]: Coveralls badge at bottom.

**Comment by PritishC** [comment]
LGTM

**Comment by prakashpp** [comment]
Already renamed to rst.

**Pull request closed without merging**